packages/iov-core/src/multichainsigner.spec.ts

```diff
@@ -173,6 +174,80 @@ describe("MultiChainSigner", () => {
 
       signer.shutdown();
     });
+
+    it("can add two chains and send on both of them", async () => {
+      pendingWithoutBnsd();
+      pendingWithoutEthereum();
+
+      const profile = new UserProfile();
+      const signer = new MultiChainSigner(profile);
+      expect(signer.chainIds().length).toEqual(0);
+
+      await signer.addChain(bnsConnector(bnsdTendermintUrl));
+      await signer.addChain(ethereumConnector(httpEthereumUrl, undefined));
+      const [bnsId, ethereumChainId] = signer.chainIds();
+
+      // Create sender identities
+      const { mainWalletId, faucet: bnsFaucet } = await addWalletWithFaucet(profile, bnsId);
+      const secpWallet = profile.addWallet(
+        Secp256k1HdWallet.fromMnemonic(
+          "oxygen fall sure lava energy veteran enroll frown question detail include maximum",
+        ),
+      );
+      const ganacheMainIdentity = await profile.createIdentity(
+        secpWallet.id,
+        ethereumChainId,
+        HdPaths.ethereum(0),
+      );
+
+      {
```

Comment:
Are these blocks just for visual delimitation or is there some other purpose?

Reply:
Visual and for scoping of the variables. I tend to do it when I want to make sure the variables are not accessed anymore afterwards and to be able to reuse the varable names